kedro-datasets/setup.py

```diff
@@ -15,54 +15,50 @@ def _collect_requirements(requires):
+databricks_require = {"databricks.ManagedTableDataset": [SPARK, PANDAS, DELTA]}
 geopandas_require = {
-    "geopandas.GeoJSONDataSet": ["geopandas>=0.6.0, <1.0", "pyproj~=3.0"]
+    "geopandas.GeoJSONDataset": ["geopandas>=0.6.0, <1.0", "pyproj~=3.0"]
 }
 holoviews_require = {"holoviews.HoloviewsWriter": ["holoviews~=1.13.0"]}
 huggingface_require = {
     "huggingface.HFDataset": ["datasets", "huggingface_hub"],
     "huggingface.HFTransformerPipelineDataset": ["transformers"],
 }
 matplotlib_require = {"matplotlib.MatplotlibWriter": ["matplotlib>=3.0.3, <4.0"]}
-networkx_require = {"networkx.NetworkXDataSet": ["networkx~=2.4"]}
+networkx_require = {"networkx.NetworkXDataset": ["networkx~=2.4"]}
 pandas_require = {
-    "pandas.CSVDataSet": [PANDAS],
-    "pandas.ExcelDataSet": [PANDAS, "openpyxl>=3.0.6, <4.0"],
-    "pandas.DeltaTableDataSet": [PANDAS, "deltalake>=0.10.0"],
-    "pandas.FeatherDataSet": [PANDAS],
-    "pandas.GBQTableDataSet": [PANDAS, "pandas-gbq>=0.12.0, <0.18.0"],
-    "pandas.GBQQueryDataSet": [PANDAS, "pandas-gbq>=0.12.0, <0.18.0"],
-    "pandas.HDFDataSet": [
+    "pandas.CSVDataset": [PANDAS],
+    "pandas.ExcelDataset": [PANDAS, "openpyxl>=3.0.6, <4.0"],
+    "pandas.DeltaTableDataset": [PANDAS, "deltalake>=0.10.0"],
+    "pandas.FeatherDataset": [PANDAS],
+    "pandas.GBQTableDataset": [PANDAS, "pandas-gbq>=0.12.0, <0.18.0"],
+    "pandas.GBQQueryDataset": [PANDAS, "pandas-gbq>=0.12.0, <0.18.0"],
+    "pandas.HDFDataset": [
         PANDAS,
         "tables~=3.6.0; platform_system == 'Windows'",
         "tables~=3.6, <3.9; platform_system != 'Windows' and python_version<'3.9'",
         "tables~=3.6; platform_system != 'Windows' and python_version>='3.9'",
     ],
-    "pandas.JSONDataSet": [PANDAS],
-    "pandas.ParquetDataSet": [PANDAS, "pyarrow>=6.0"],
-    "pandas.SQLTableDataSet": [PANDAS, "SQLAlchemy>=1.4, <3.0"],
-    "pandas.SQLQueryDataSet": [PANDAS, "SQLAlchemy>=1.4, <3.0", "pyodbc~=4.0"],
-    "pandas.XMLDataSet": [PANDAS, "lxml~=4.6"],
-    "pandas.GenericDataSet": [PANDAS],
+    "pandas.JSONDataset": [PANDAS],
+    "pandas.ParquetDataset": [PANDAS, "pyarrow>=6.0"],
+    "pandas.SQLTableDataset": [PANDAS, "SQLAlchemy>=1.4, <3.0"],
+    "pandas.SQLQueryDataset": [PANDAS, "SQLAlchemy>=1.4, <3.0", "pyodbc~=4.0"],
+    "pandas.XMLDataset": [PANDAS, "lxml~=4.6"],
+    "pandas.GenericDataset": [PANDAS],
 }
-pickle_require = {"pickle.PickleDataSet": ["compress-pickle[lz4]~=2.1.0"]}
-pillow_require = {"pillow.ImageDataSet": ["Pillow~=9.0"]}
+pickle_require = {"pickle.PickleDataset": ["compress-pickle[lz4]~=2.1.0"]}
+pillow_require = {"pillow.ImageDataset": ["Pillow~=9.0"]}
 plotly_require = {
-    "plotly.PlotlyDataSet": [PANDAS, "plotly>=4.8.0, <6.0"],
-    "plotly.JSONDataSet": ["plotly>=4.8.0, <6.0"],
+    "plotly.PlotlyDataset": [PANDAS, "plotly>=4.8.0, <6.0"],
+    "plotly.JSONDataset": ["plotly>=4.8.0, <6.0"],
 }
 polars_require = {
-    "polars.CSVDataSet": [POLARS],
-    "polars.GenericDataSet":
```

Comment:
What happened to `polars.GenericDataSet`?

Reply:
It's replaced by `polars.EagerPolarsDataset`.

Reply:
Right, I see it's in the PR description too 🤦 😂